exp/controllers/azuremachinepool_controller.go

```diff
@@ -158,19 +157,16 @@ func (ampr *AzureMachinePoolReconciler) SetupWithManager(ctx context.Context, mg
 
 // Reconcile idempotently gets, creates, and updates a machine pool.
 func (ampr *AzureMachinePoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_ ctrl.Result, reterr error) {
+	ctx, span := tele.Tracer().Start(
```

Comment:
I'd hate to lose the trace attributes that were added to this span. Please consider reverting to the previous Start.

Reply:
sorry about that! re-added in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/1460/commits/f21e1c8a0c67c16b223d2d3e3ac57d00706a36a1